packages/app/test/cypress/integration/20-basic-features/access-to-pagelist.spec.ts

```diff
@@ -86,10 +90,18 @@ context('Access to timeline', () => {
       cy.get('.nav-title > li').eq(1).find('a').click();
       cy.get('button.close').eq(0).click();
     });
-    cy.screenshot(`${ssPrefix}2-timeline-list-fullscreen`, {capture: 'viewport'});
+
+    // Hide release & license on home page , page-list count and page creation & date
+    cy.screenshot(`${ssPrefix}2-timeline-list-fullscreen`, {
+      capture: 'viewport',
+      blackout: ['.card-timeline:eq(0) > .card-body > #wiki > p > a > img']
+    });
```

Comment:
This one looks like the whole content is changed and I don't know what are being blackouted here.

Before
<img width="1095" alt="image" src="https://user-images.githubusercontent.com/35527421/189107600-185e080d-8fd7-41fa-bc1f-adb89a35f715.png">

After
<img width="1077" alt="image" src="https://user-images.githubusercontent.com/35527421/189107663-bee51991-ad50-4f39-9caf-e0ce7909498c.png">